src/tabs/tab-panel.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+
+import type {TabPropsT} from './types.js';
+
+class TabPanel extends React.Component<TabPropsT> {
+  static defaultProps: $Shape<TabPropsT> = {
+    disabled: false,
+    onSelect: () => {},
+    onClick: () => {},
+    onKeyDown: () => {},
+    title: '',
+  };
```

Comment:
Should we have a `render` here?

Reply:
huh, yeah not quite sure how this is working

Reply:
Do we need one? I only added this 'component' as a way to provide an API to accept props together for the tabs and the associated panels.

Reply:
Added 
``
render() {
    return null;
  }
``